packages/cli/test/sim/multi_fork.test.ts

```diff
@@ -101,17 +102,34 @@ const env = await SimulationEnvironment.initWithDefaults(
       validator: {
         type: ValidatorClient.Lodestar,
         options: {
+          // this builder selection will make it use produceBlockV2 and respond with full block
           clientOptions: {
             useProduceBlockV3: false,
-            // this builder selection will make it use produceBlockV2
             "builder.selection": "executiononly",
           },
         },
       },
       execution: ExecutionClient.Nethermind,
       keysCount: 32,
     },
-    {id: "node-4", beacon: BeaconClient.Lighthouse, execution: ExecutionClient.Geth, keysCount: 32},
+    {
+      id: "node-4",
+      beacon: BeaconClient.Lodestar,
+      validator: {
+        type: ValidatorClient.Lodestar,
+        options: {
+          // this builder selection will make it use produceBlindedBlockV2 and respond with blinded version
+          // of local block and subsequent publishing via publishBlindedBlock
+          clientOptions: {
+            useProduceBlockV3: false,
+            "builder.selection": "maxprofit",
```

Comment:
need to set `blindedLocal: true` here?

Reply:
no this doesn't use produceBlockV3 flow but calls older method of `produceBlindedBlockV2` which also blinds local and return it in the blinded fashion just like produceBlockV3 does but with less metadata in the response